crates/api_models/src/refunds.rs

```diff
@@ -132,29 +132,25 @@ pub struct RefundListRequest {
     pub payment_id: Option<String>,
     /// Limit on the number of objects to return
     pub limit: Option<i64>,
-    /// The time at which refund is created
-    #[serde(default, with = "custom_serde::iso8601::option")]
-    pub created: Option<PrimitiveDateTime>,
-    /// Time less than the refund created time
-    #[serde(default, rename = "created.lt", with = "custom_serde::iso8601::option")]
-    pub created_lt: Option<PrimitiveDateTime>,
-    /// Time greater than the refund created time
-    #[serde(default, rename = "created.gt", with = "custom_serde::iso8601::option")]
-    pub created_gt: Option<PrimitiveDateTime>,
-    /// Time less than or equals to the refund created time
-    #[serde(
-        default,
-        rename = "created.lte",
-        with = "custom_serde::iso8601::option"
-    )]
-    pub created_lte: Option<PrimitiveDateTime>,
-    /// Time greater than or equals to the refund created time
-    #[serde(
-        default,
-        rename = "created.gte",
-        with = "custom_serde::iso8601::option"
-    )]
-    pub created_gte: Option<PrimitiveDateTime>,
+    /// The starting point within a list of objects
+    pub offset: Option<i64>,
+    /// The time range for which objects are needed. TimeRange has two fields start_time and end_time from which objects can be filtered as per required scenarios (created_at, time less than, greater than etc).
+    pub time_range: Option<TimeRange>,
+    /// The List of connector filters for objects list
+    pub connector: Option<Vec<String>>,
+    /// The List of currency filters for objects list
```

Comment:
```suggestion
    /// The List of currencies to filter refunds list
```
Update comment phrasing & remove ambiguity from the rest of the comments as well